app/views/include/forms2/textarea.scala.html

```diff
@@ -27,15 +27,45 @@
     rows: String = "",
     maxLength: String = "",
     charCount: Boolean = false,
-    largeWidth: Boolean = false
+    largeWidth: Boolean = false,
+    heading: String = "",
+    section: String = ""
 )(implicit lang:Lang,  m: Messages)
 
+@labelSpan = {
+    @if(labelText.nonEmpty) {
+        @if(labelHidden) {<span class="visuallyhidden">}
+            @Messages(labelText)
+        @if(labelHidden){</span>}
+    }
+}
+
+@label = {
+    @if(heading.nonEmpty) {
+        <div class="page-header">
+            <h1>
+                <label class="heading-xlarge" for="@field.id">
+                    @Messages(heading)
+                </label>
+            </h1>
+            @if(section.nonEmpty) {
+                <p class="heading-secondary">
+                    <span class="visuallyhidden">This section is: </span>@Messages(section)
+                </p>
+            }
+        </div>
+        <span class="form-label">
```

Comment:
<p>@labelSpan</p>

This is my fault, and is what Martin mentioned in the session earlier